### PR TITLE
Enabled operations time comparison in CI

### PR DESCRIPTION
**Pull request by yashashreesuresh**
This commit enables the comparison of time taken for individual
checks/operations in PR against gold values in CI and displays the
percentage change.

Fixes: #81

### CODE REVIEW

**Comment by yashashreesuresh** [comment]
Please have a look @chaitanyaenr

File: test.sh [hunk 1]
```diff
@@ -62,6 +62,24 @@ do
...
+  echo 'Check | Gold time (s) | PR time (s) | Delta (PR time - Gold time)' > results.markdown
+  echo '------|---------------|-------------|----------------------------' >> results.markdown
+  checks_in_pr=`jq '.Average | keys | .[]' test_daemon_disabled/time_tracker.json`
+  checks_in_master=`jq '.Average | keys | .[]' time_tracker.json`
+  for check in $checks_in_pr; do
+    pr_time=$(jq -r ".Average[$check]" test_daemon_disabled/time_tracker.json);
+    if [[ `echo $checks_in_master | grep -w $check` ]];
+    then
+      gold_time=$(jq -r ".Average[$check]" time_tracker.json);
+      delta=$(bc -l <<<"scale=6; (${pr_time}-${gold_time})/1")
```
Comment (chaitanyaenr) [inline test.sh:75]: Let's display the delta in percentage i.e percentage time increase/decrease when compared to the master branch.

**Comment by paigerube14** [comment]
LGTM

**Pull request merged**